rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -1940,6 +1940,13 @@
                </property>
               </widget>
              </item>
+             <item>
+              <widget class="QCheckBox" name="emptyHdd0Tmp">
+               <property name="text">
+                <string>Empty /dev_hdd0/tmp/</string>
+               </property>
+              </widget>
+             </item>
```

Comment:
This leads to a horrible layout

Reply:
![image](https://github.com/user-attachments/assets/127cc875-bf97-415a-b72a-21ffeb921e31)
I don't follow even after using the QT editor to verify that I typed the code correctly.
Looks great on all the GUI stylesheets, and functions in global and custom configurations.
All I can think of is wanting empty dev_hdd before enable host root. Just to appear better Alphabetically sorted.

Reply:
I don't know what you're looking at, but it makes the two group boxes next to it huge.

Reply:
![image](https://github.com/user-attachments/assets/0239a24b-565b-46fe-96cf-cfbba486f251)
Looks the same as before my PR to me.

Reply:
It's obviously not

Reply:
I can't edit something I don't see

Reply:
I ran it locally and it is clearly larger than the boxes below for example.
Now add another checkbox and it will look hilarious.

Reply:
It's a bit annoying to edit.
But I guess I can try to fix the layout.

Reply:
I see the system tab isn't like the core tab with the layout left, middle, and right. I can add that.

Reply:
you could just cherrypick this:
https://github.com/RPCS3/rpcs3/commit/7750f1398832f7c26145e0170d6bb7a4ae4ed175
(or do it all again if you want to practise)

Reply:
![image](https://github.com/user-attachments/assets/a6f0ed80-bd74-4e1d-a371-4e099eaea6f3)
Your changes look flawless and it appears I did end up seeing the layout issue 20 minutes too late.